metadata-ingestion/src/datahub/sql_parsing/sqlglot_utils.py

```diff
@@ -90,7 +97,31 @@ def _simplify_node_expressions(node: sqlglot.exp.Expression) -> None:
         # Replace all literals in the expressions with a single placeholder.
         is_last_literal = True
         for i, expression in reversed(list(enumerate(node.expressions))):
-            if isinstance(expression, sqlglot.exp.Literal):
+            #
+            # sqlglot has a bug related to T-SQL dialect
+            # for example, this statement using double quotes
+            #
+            # INSERT INTO TBL
+            # VALUES("str1", 'str2', 1)
+            #
+            # will be parsed like this:
+            #
+            # INSERT INTO TBL
+            # VALUES(COLUMN, LITERAL, LITERAL)
+            #
+            # as result it will be generalized as
+            #
+            # INSERT INTO TBL
+            # VALUES("str1", ?, ?)
+            #
+            # instead of
+            #
+            # INSERT INTO TBL
+            # VALUES(?, ?, ?)
+            #
+            # adding sqlglot.exp.Column to the condition fixes it
+            #
+            if isinstance(expression, (sqlglot.exp.Literal, sqlglot.exp.Column)):
```

Comment:
I'm not sure this is actually a safe change

if I have `select col1 from mytbl` and `select col2 from mytbl`, we want those to have different fingerprints. With this change, they'll end up with the same fingerprint

Reply:
rolled it back, because it's better to fix it in T-SQL dialect definition of sqlglot library, than making the crutches here
https://github.com/datahub-project/datahub/pull/9838/commits/7b3126f1fc0e0fc3f9caf576abc71df5c89ac49f

it might have been already fixed there
https://github.com/tobymao/sqlglot/commit/d31ae0decb46678851744356c7b113f8c1c3e8c9